src/components/Homepage/LendByCategory/FeaturedLoansCarousel.vue

```diff
@@ -1,14 +1,23 @@
 <template>
 	<div>
 		<h3>
-			<a :href="currentSlideCategory.url">
+			<a
+				:href="currentSlideCategory.url"
+				v-kv-track-event="[
+					'homepage',
+					'click-hero-category',
+					cleanCategoryLink(currentSlideCategory),
+					'true']"
```

Comment:
Please remove the 4th parameter `true` from all of these. This param needs to either be a number or not present, string `true` will just evaluate to NaN see `src/plugins/kv-analytics-plugin.js:78`

I know they've been copy and pasted around the codebase quite a bit, I'm guilty of it too